pegasus/sites.v3/code.org/public/teacher-dashboard/section_progress.haml

```diff
@@ -73,6 +73,9 @@ content-type: text/ng-template
       %dt
         %span.level_link.perfect &nbsp;
       %dd= I18n.t('dashboard_progress_completed_perfect')
+      %dt
+        %span.level_link.submitted &nbsp;
+      %dd= I18n.t('dashboard_progress_submitted')
```

Comment:
@tanyaparker any idea how I get this string to work?  Presuming it should be generated from the new addition to en.yml just above.  Tried a full build, even.